src/evaluate.ts

```diff
@@ -232,9 +232,11 @@ async function evaluateCode(
     }
 }
 
-async function evaluateSelection(document: {}, options) {
+async function evaluateSelection(document = {}, options) {
     const doc = util.getDocument(document);
-    const selectionFn: Function = options.selectionFn;
+    const selectionFn: (
+        editor: vscode.TextEditor
+    ) => [vscode.Selection, string] = options.selectionFn;
```

Comment:
Using the Function type gives us no real type safety and so it's better to specify the actual types:

https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/ban-types.md